elliottlib/bzutil.py

```diff
@@ -1058,6 +1058,29 @@ def get_highest_security_impact(bugs):
     return 'Low'
 
 
+def compare_impact(bug_a, bug_b):
+    """
+    Check if CVE priority of bug_a if lower than bug_b
+    first compare severity level, if severity level is the same
+    then compare CVE number
+    """
+    buga_l = constants.security_impact_map[get_highest_security_impact([bug_a])]
+    bugb_l = constants.security_impact_map[get_highest_security_impact([bug_b])]
+    if buga_l == bugb_l:
+        return int(bug_a.alias[0].replace("CVE", "").replace("-", "")) > int(bug_b.alias[0].replace("CVE", "").replace("-", ""))
+    else:
+        return buga_l < bugb_l
+
+
+def sort_cve_bugs(bugs):
```

Comment:
Consider implementing with builtin sort functions instead of homebrewed bubble sort. https://docs.python.org/3/howto/sorting.html

Reply:
updated